README.md

```diff
@@ -121,9 +121,7 @@ On every _Timestamp_ cache update, `hazelcast-hibernate` publishes an invalidati
 
 Local region cache eviction can be configured using two parameters:
  - _time-to-live_ - defining the lifespan of cache entries (defaults to 1 hour)
- - -eviction-size_ - defining the maximum cache size (defaults to 100000)
-
-When maximum size is hit, 20% of the entries will be evicted automatically.
+ - _eviction-size_ - defining the maximum cache size (defaults to 100000)
```

Comment:
Seems like both ttl and size default to Integer.MAX_VALUE due to the default map configuration of Hazelcast. The max value of 100_000 takes action when a LocalRegion is built where hz instance is null. And I believe we discussed a while ago if the instantiation with null hz instance is even possible :no_mouth: